config/initializers/simple_token_authentication.rb

```diff
@@ -1,4 +1,4 @@
-SimpleTokenAuthentication.configure do |config|
+SimpleTokenAuthentication.configure do |_config|
```

Comment:
Better to comment this file completely.

When you will need to add configuration inside `configure do |_config|` you will rename `_config` back to `config`.